src/components/tabs/components/tab-dropdown-controls/tab-dropdown-controls.module.css

```diff
@@ -1,6 +1,21 @@
 .selectRoot {
   position: relative;
   width: 100%;
+
+  /* In nested tabs, include an underline-ish style under the select */
+  &.isNested {
+    margin-bottom: 4px;
+
+    &::after {
+      background: var(--token-color-border-primary);
+      bottom: -4px;
+      content: '';
+      height: 1px;
+      left: 0;
+      position: absolute;
+      right: 0;
+    }
+  }
```

Comment:
❓ Is it possible to use a normal CSS border here instead of a pseudo element?

Reply:
Great question - I don't think so, at least not in the way I've gone about things! Added a comment to clarify why.